python/dgl/data/graph_serialize.py

```diff
@@ -115,16 +124,18 @@ def load_graphs(filename, idx_list=None):
     Parameters
     ----------
     filename: str
-        filename to load DGLGraphs
+        filename to load graphs
     idx_list: list of int
         list of index of graph to be loaded. If not specified, will
         load all graphs from file
 
     Returns
     ----------
-    graph_list: list of immutable DGLGraphs
+
+    graph_list: list of DGLGraphs / DGLHeteroGraph
+
     labels: dict of labels stored in file (empty dict returned if no
-    label stored)
+    label stored). Always the full data regardless of the idx_list

```

Comment:
This will not correctly render. Change to the following
```python
Returns
--------
graph_list: list of DGLGraphs / DGLHeteroGraph
    The loaded graphs.
labels: dict[str, Tensor]
    The graph labels stored in file. If no label is stored, the dictionary is empty.
    Regardless of whether the ``idx_list`` argument is given or not, the returned dictionary
    always contains labels of all the graphs.
```